internal/cmd/validate.go

```diff
@@ -151,115 +165,116 @@ func validateCmdFunc(cmd *cobra.Command, filenames []string) error {
 			}
 
 			// Output errors
-			outputDeveloperErrorsWithLineOffset(validateContents, devErrs.InputErrors, schemaOffset)
+			outputDeveloperErrorsWithLineOffset(toPrint, validateContents, devErrs.InputErrors, schemaOffset)
```

Comment:
Not sure how to hit this code path yet

Reply:
IIRC this and the other are some combination of compilation errors and typesystem errors. Grepping for `DeveloperErr` and `ErrorWithSource` should probably turn something up.

Reply:
JK you've already got some of that kind of thing in there...